api/models/Tag.js

```diff
@@ -294,6 +294,24 @@ module.exports = bookshelf.Model.extend({
     })
   },
 
+  taggedPostCount: tagId => {
+    return bookshelf.knex('posts_tags')
```

Comment:
@levity should we be expanding this query, and another similar one in CommunityTag to filter for only `active` posts?